index.html

```diff
@@ -0,0 +1,68 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Friends of Homeless Animals</title>
+  <link href="http://fonts.googleapis.com/css?family=Roboto+Slab:400,300,700&subset=latin,greek" rel="stylesheet" type="text/css">
+  <link rel="stylesheet" href="css/reset.css">
+  <link rel="stylesheet" href="css/main-styling.css">
+</head>
+<body>
+
+  <header id="page-header">
+    <h1><a href="index.html">Friends of Homeless Animals</a></h1>
+    <nav>
+      <ul>
+        <li><strong><a href="index.html">Home</a></strong></li>
+        <li><strong><a href="about.html">About</a></strong></li>
+        <li><strong><a href="store.html">Resale Shop</a></strong></li>
+        <li><strong><a href="contact.html">Contact</a></strong></li>
+      </ul>
+    </nav>
+  </header>
+
+  <section class="banner">
+    <!--<a href="insert rotating image"></a>-->
+  </section>
+
+  <div class="wrapper">
+    <section class="buttons">
+      <section class="actions">
+        <header>
+          <h3>Adopt</h3>
+        </header>
+        <p>Interested in adopting one of our dogs?</p>
+        <a class="button" href="adopt.html">Adopt a Dog</a>
+      </section>
+      <section class="actions">
+        <header>
+          <h3>Volunteer</h3>
+        </header>
+        <p>Want to spend time with our pups?</p>
+        <a class="button" href="volunteer.html">Volunteer Opportunities</a>
+        </section>
+      <section class="actions">
+        <header>
+          <h3>Foster</h3>
+        </header>
+          <p>Looking to foster one our pups?</p>
+          <a class="button" href="foster.html">Foster Opportunities</a>
+      </section>
+      <section class="actions">
+        <header>
+          <h3>Donate</h3>
+        </header>
+          <p>Would you like to donate or sponsor a dog?</p>
+          <a class="button" href="donate.html">Donate/Sponsor</a>
+      </section>
+    </section>
+  </div>
+
+  <footer>
+    <div class="wrapper">
+      <p>Sign up for our weekly newsletter for shelter updates, photos, stories, etc.</p>
+    </div>
+  </footer>
+
+</body>
+</html>
```

Comment:
Great job! Really good start. Start solidifying that layout, try to make it responsive, and link your pages together.